packages/cozy-stack-client/src/JobCollection.js

```diff
@@ -48,13 +57,36 @@ class JobCollection {
 
   /**
    * Return a normalized job, given its id
+   *
+   * @param {string} id - id of the job
+   *
+   * @returns {JobDocument}
    */
   async get(id) {
     return Collection.get(this.stackClient, uri`/jobs/${id}`, {
       normalize: normalizeJob
     })
   }
 
+  /**
+   * Update the job's state
+   * This does work only for jobs comming from @client triggers
+   *
+   * @param {JobDocument} job - io.cozy.jobs document
+   */
+  async update(job) {
```

Comment:
I'm wondering if this method should not check if `job.attributes.worker === client` before making the call. Like that, it's explicit that only this worker is supported. Should throw an error if not. WDYT?

Reply:
I wondered too, but chose to let this check to the stack. This avoids some later error if the stack changes.
But since you asked, I tested the stack behavior and it sends a `403 - {"error":"Forbidden"}`

Let's be more explicit and throw with a more explicit message